packages/nuekit/src/create.js

```diff
@@ -24,7 +24,10 @@ async function serve(args) {
 }
 
 export async function create(args = {}) {
-  if (!args.name) args.name = args.paths.shift().split('/').filter(Boolean).join('/') || 'simple-blog'
+  if (!args.name) {
+    const first_path = args.paths.shift()
+    args.name = first_path ? first_path.split('/').filter(Boolean).join('/') : 'simple-blog'
+  } 
```

Comment:
```suggestion
  if (!args.name) args.name = args.paths.shift()?.split('/')?.filter(Boolean)?.join('/') || 'simple-blog'
```

What do you think about this? Or is it not readable enough?

Reply:
looks good to me 👍